.nais/prod-gcp.yml

```diff
@@ -84,7 +85,9 @@ spec:
     - name: PDL_URL
       value: https://pdl-api.prod-fss-pub.nais.io
     - name: PEN_URL
-      value: https://pensjon-selvbetjening-fss-gateway.prod-fss-pub.nais.io
+      value: https://pensjon-pen.prod-fss-pub.nais.io
```

Comment:
Jeg mener at det ikke lenger skal være behov for å kalle disse pub-urlene. Altså at man kan kalle intern-url fra gcp-fss. Har du lyst til å sjekke det ut?

Reply:
Hmm, det visste jeg ikke.. Kan godt sjekke ut.

Reply:
Glem det jeg skrev. Pen har ikke intern-ingress